unordered updates (#1694)
cpinitiative/usaco-guide: content/4_Gold/Unordered.mdx

meaningful way, so traversals of an unordered set will return elements in some
arbitrary order.

Oftentimes, `unordered_map` and `unordered_set` can be used as a drop-in
replacement for their `map` and `set` counterparts, as long as the key does not require custom hashing.

The operations for `unordered_map` include:

- `insert`, `erase`, `count`, which all function the same as

>   [sorted sets](/bronze/intro-sets)

Reviewer: add period

meaningful way, so traversals of an unordered set will return elements in some
arbitrary order.

> Oftentimes, `unordered_map` and `unordered_set` can be used as a drop-in
> replacement for their `map` and `set` counterparts, as long as the type being
> hashed does not require custom hashing.

Reviewer: ```suggestion
Oftentimes, `unordered_map` and `unordered_set` can be used as a drop-in
replacement for their `map` and `set` counterparts, as long as the key does not require custom hashing.
```

The operations for `unordered_map` include:

> - `insert`, `erase`, `count`, which are the same as

Reviewer: ```suggestion
- `insert`, `erase`, `count`, which function the same as
```

  [sorted sets](/bronze/intro-sets)

> - `find`, which gets the iterator to an element.

Reviewer: ```suggestion
- `find`, which returns the iterator to an element.
```
